website/static/data/events.json

```diff
@@ -1,5 +1,9 @@
 [   
   {
+    "title": "Biweekly Report (Jul 1 - Jul 30)",
```

Comment:
I think you mean July 15th here.
```suggestion
    "title": "Biweekly Report (Jul 1 - Jul 15)",
```

Reply:
<img width="1505" alt="image" src="https://user-images.githubusercontent.com/36651058/180179123-836999fc-231f-4d79-aa5f-3d2cac600565.png">